src/dfastbe/bank_erosion.py

```diff
@@ -607,7 +622,9 @@ def _process_discharge_levels(
                     if self.debug:
                         bcrds_mid = (bcrds[:-1] + bcrds[1:]) / 2
                         bank_coords_points = [Point(xy1) for xy1 in bcrds_mid]
-                        bank_coords_geo = GeoSeries(bank_coords_points)
+                        bank_coords_geo = GeoSeries(
+                            bank_coords_points, crs="EPSG:28992"
```

Comment:
please use the crs from the configfile class , check how you can pass the crs from the configfile to here